ext/fs/std_fs.rs

```diff
@@ -371,7 +372,7 @@ impl FileSystem for RealFs {
     &self,
     path: &Path,
     access_check: Option<AccessCheckCb>,
-  ) -> FsResult<Vec<u8>> {
+  ) -> FsResult<Cow<'static, [u8]>> {
```

Comment:
Ah, I can get this to work if I return a v8 string instead. Will update this pr.